.gitignore

```diff
@@ -3,3 +3,7 @@
 /build/
 /dist/
 /git-multimail/*.egg-info
+
+# The Vi/Vim temporary files
+*.swp
```

Comment:
This is not a project-specific file. These ignore rules should go to your user-wide `.gitignore` (`$HOME/.config/git/ignore`).